From 4ada76062e20b7620a1a7790f88a50d916ee0d88 Mon Sep 17 00:00:00 2001
From: Mark Giaconia <markg@apache.org>
Date: Thu, 6 Feb 2014 01:11:37 +0000
Subject: [PATCH] OPENNLP-645 Just trying to quickly stabilize the build.
 Please review ASAP.

I changed line 208 of uima.chunker.ChunkerTrainer to this:
   ChunkerModel chunkerModel = ChunkerME.train(language, ObjectStreamUtils.createObjectStream(mChunkSamples), ModelUtil.createDefaultTrainingParameters(), ChunkerFactory.create(null));

and line 170 in uima.sentecedetect.sentencedetectertrainer to:
  TrainingParameters mlParams = ModelUtil.createDefaultTrainingParameters();

git-svn-id: https://svn.apache.org/repos/asf/opennlp/trunk@1565030 13f79535-47bb-0310-9956-ffa450edef68
---
 .../src/main/java/opennlp/uima/chunker/ChunkerTrainer.java  | 6 ++++--
 .../opennlp/uima/sentdetect/SentenceDetectorTrainer.java    | 4 ++--
 2 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/opennlp-uima/src/main/java/opennlp/uima/chunker/ChunkerTrainer.java b/opennlp-uima/src/main/java/opennlp/uima/chunker/ChunkerTrainer.java
index 2d2f4a424..c38cd3a17 100644
--- a/opennlp-uima/src/main/java/opennlp/uima/chunker/ChunkerTrainer.java
+++ b/opennlp-uima/src/main/java/opennlp/uima/chunker/ChunkerTrainer.java
@@ -24,10 +24,12 @@
 import java.util.List;
 
 import opennlp.tools.chunker.ChunkSample;
+import opennlp.tools.chunker.ChunkerFactory;
 import opennlp.tools.chunker.ChunkerME;
 import opennlp.tools.chunker.ChunkerModel;
 import opennlp.tools.ml.maxent.GIS;
 import opennlp.tools.util.ObjectStreamUtils;
+import opennlp.tools.util.model.ModelUtil;
 import opennlp.uima.util.CasConsumerUtil;
 import opennlp.uima.util.ContainingConstraint;
 import opennlp.uima.util.OpennlpUtil;
@@ -203,8 +205,8 @@ public void collectionProcessComplete(ProcessTrace trace)
       throws ResourceProcessException, IOException {
     GIS.PRINT_MESSAGES = false;
     
-    ChunkerModel chunkerModel = ChunkerME.train(language, ObjectStreamUtils.createObjectStream(mChunkSamples), 100, 5);
-    
+    ChunkerModel chunkerModel = ChunkerME.train(language, ObjectStreamUtils.createObjectStream(mChunkSamples), ModelUtil.createDefaultTrainingParameters(), ChunkerFactory.create(null));
+  
     // dereference to allow garbage collection
     mChunkSamples  = null;
     
diff --git a/opennlp-uima/src/main/java/opennlp/uima/sentdetect/SentenceDetectorTrainer.java b/opennlp-uima/src/main/java/opennlp/uima/sentdetect/SentenceDetectorTrainer.java
index a71aa6130..cbc7f377c 100644
--- a/opennlp-uima/src/main/java/opennlp/uima/sentdetect/SentenceDetectorTrainer.java
+++ b/opennlp-uima/src/main/java/opennlp/uima/sentdetect/SentenceDetectorTrainer.java
@@ -167,8 +167,8 @@ public void collectionProcessComplete(ProcessTrace trace)
     SentenceDetectorFactory sdFactory = SentenceDetectorFactory.create(
             null, language, true, null, eos);
     
-    TrainingParameters mlParams = ModelUtil.createTrainingParameters(100, 5);
-    
+  //  TrainingParameters mlParams = ModelUtil.createTrainingParameters(100, 5);
+     TrainingParameters mlParams = ModelUtil.createDefaultTrainingParameters();
     ObjectStream<SentenceSample> samples = ObjectStreamUtils.createObjectStream(sentenceSamples);
     
     Writer samplesOut = null;
